tests/php/Control/HTTPRequestTest.php

```diff
@@ -25,6 +25,21 @@ public function testMatch()
         $this->assertEquals(array("_matched" => true), $request->match('add', true));
     }
 
+    /**
+     * @useDatabase false
+     */
+    public function testWildCardMatch()
+    {
+        $request = new HTTPRequest('GET', 'admin/crm/test');
+        $this->assertEquals(['$1' => 'crm', '$2' => 'test'], $request->match('admin/$@', true));
+        $this->assertTrue($request->allParsed());
+
+        $request = new HTTPRequest('GET', 'admin/crm/test');
+        $this->assertEquals(['_matched' => true], $request->match('admin/$*', true));
+        $this->assertTrue($request->allParsed());
+        $this->assertEquals('crm/test', $request->remaining());
+    }
```

Comment:
Should we add some less happy-path tests, here? Maybe some URLs that use both named parameters _and_ variadics and wildcards?

Reply:
Yep, that's probably a good idea - I'll add a few more